src/blot-formatters/actions/image-resize-action.ts

```diff
@@ -0,0 +1,76 @@
+import { ResizeAction } from "quill-blot-formatter";
+
+/**
+ * Custom resize action to disable text selection
+ * while dragging image resize handles
+ */
+export class ImageResizeAction extends ResizeAction {
+    // Can't use super.onDrag here
+    // because it's defined as a property
+    // on the superclass, instead of as a method,
+    // so the logic is copied from the superclass here
+    onDrag = (event: MouseEvent) => {
+        if (!this.formatter.currentSpec) {
+            return;
+        }
+
+        const target = this.formatter.currentSpec.getTargetElement();
+        if (!target) {
+            return;
+        }
+
+        ImageResizeAction.toggleUserSelect(false);
+
+        const deltaX = event.clientX - this.dragStartX;
+        let newWidth: number;
+
+        if (
+            this.dragHandle === this.topLeftHandle ||
+            this.dragHandle === this.bottomLeftHandle
+        ) {
+            newWidth = Math.round(this.preDragWidth - deltaX);
+        } else {

```

Comment:
Could this `else` block be removed by initializing `newWidth` to `Math.round(this.preDragWidth + deltaX);` on line 25?

Reply:
Yep, good call.